bench/wtperf/wtperf_misc.c

```diff
@@ -141,6 +141,8 @@ backup_read(WTPERF *wtperf, WT_SESSION *session)
         /* Open the file handle. */
         testutil_snprintf(buf, WT_BACKUP_COPY_SIZE, "%s/%s", wtperf->home, filename);
         error_sys_check(rfd = open(buf, O_RDONLY, 0644));
+        if (rfd == -1)
```

Comment:
Given the specific Coverity warning, I wonder if it might be better to change the condition to {{>= 0}} to eliminate future warnings?

Reply:
Fixed.